testsuite/manualmode/src/test/java/org/jboss/as/test/manualmode/management/persistence/AbstractGitRepositoryTestCase.java

```diff
@@ -82,7 +82,7 @@ protected void closeEmptyRemoteRepository() throws Exception {
         if (emptyRemoteRepository != null) {
             emptyRemoteRepository.close();
         }
-        FileUtils.delete(emptyRemoteRoot.getParent().toFile(), FileUtils.RECURSIVE | FileUtils.RETRY);
+        FileUtils.delete(emptyRemoteRoot.getParent().toFile(), FileUtils.RECURSIVE | FileUtils.RETRY | FileUtils.SKIP_MISSING | FileUtils.IGNORE_ERRORS);
```

Comment:
`FileUtils.IGNORE_ERRORS`, do we really want to silently ignore any errors if the file is not deleted?I guess allowing to fail in these cases would make it more robust and help detect unexpected code behaviors ...

Reply:
i don't know the internal of Windows file handles but sometimes the handle is not effectively released at the time we try to delete the file even if emptyRemoteRepository.close(); has been called. Also this might be environmental (like having a github desktop or IDE running with this folder open)